spec/support/admin/firm_page.rb

```diff
@@ -7,4 +7,6 @@ class Admin::FirmPage < SitePrism::Page
   element :move_advisers, '.t-move-advisers'
   element :sign_in_as_principal, '.t-sign-in-as-principal'
   element :new_adviser, '.t-new-adviser'
+  element :approved, '.t-approved'
```

Comment:
same comment as the line below, tho this one may be a bit harder, perhaps using `:last-child`?